table-api/src/main/java/io/deephaven/api/agg/AggregationPairs.java

```diff
@@ -49,6 +49,11 @@ public void visit(Count count) {
         out = Stream.empty();
     }
 
+    @Override
+    public void visit(CountWhere countWhere) {
+        out = Stream.empty();
```

Comment:
How is this used, it does seem like we legitimately have inputs and outputs.

Reply:
We do, but not pairs of input/output. We have a single output and 0..n input so this is not used.